content/en/contribute/medic/onboarding/product-team.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Product Team"
+linkTitle: "Product Team"
+weight: 3
+description: >
+  Introduction to Product Team 
+---
+
+The goal of this page is to convey a general “lay of the land” so someone starting can see a lot of what’s out there related to Product Team without having to be surprised each day as new things pop up.
+
+## About Product Team
+The Product Team as a whole is focused on digital health tools helping to achieve Universal Health Coverage, and care reaching everyone when and where they need it. 
```

Comment:
There is a broader definition of the Product Team's scope [here](https://docs.communityhealthtoolkit.org/contribute/medic/).

Reply:
Not sure if I did it right, can you please check again?